SpatialGDK/Source/SpatialGDKEditor/Public/Utils/LaunchConfigurationEditor.h

```diff
@@ -21,13 +21,19 @@ DECLARE_DELEGATE_OneParam(FOnSpatialOSLaunchConfigurationSaved, const FString&)
 	UPROPERTY(EditAnywhere, Category = "Launch Configuration")
 	FSpatialLaunchConfigDescription LaunchConfiguration;
 
+	/** Tick this if this configuration will be used for cloud deployments. */
+	UPROPERTY(EditAnywhere, Category = "Launch Configuration")
+	bool bIsCloudConfiguration = true;
```

Comment:
Is there a way we can decide this when deploying using the editor? 

What if instead we generated two configs for each setup?
a.json
a.cloud.json
and then used the appropriate one?

Reply:
Hmmmm, we certainly could generate both automatically when using the LaunchConfigGenerator window, although we'd need a way of communicating this to the user since they are the ones specifying the name of the config. Do we think it's worth the additional UX to tell users that if they want a cloud config they need to generate one and then find the .cloud. version? I'm wondering what's simpler.

Reply:
Spoke offline, current implementation is good. Was missconception that this was the cloud deploy window.